decentraland/unity-explorer
# Fix: analytics payload overflow for error message

## What does this PR change?

- add check to not overflow the message

## How to test the changes?

1. Launch the explorer
2. Check happy path works fine
